docs/api.md

```diff
@@ -4318,6 +4345,41 @@ Determines whether sink is interested in the logger with the given name and seve
 - `hints` <[Object]> optional formatting hints
   - `color` <[string]> preferred logger color
 
+### class: _Screencast
+
+A [_Screencast] instance is passed to the callback of [`browserContext._enableScreencast`](#browsercontextenablescreencastoptions-callback).
+
+An example of listening to screencasts on a context:
+```js
+const { webkit } = require('playwright');  // Or 'chromium' or 'webkit'.
+
+(async () => {
+  const browser = await webkit.launch();
+  const context = await browser.newContext();
+  await context._enableScreencast({width: 640, height: 480, dir: '/tmp/video'}, async screencast => {
+    console.log('Starting screencast: ' + screencast.page().url());
```

Comment:
Similarly to Downloads, we should not have the `page` accessor - page can be dead by now.

Reply:
Done.